### PR TITLE
Introduce the `analytics._cdn` field

### PR DESCRIPTION
**Pull request by juliofarah**
This PR introduces the `analytics._cdn` field (with the same value as
cdn host) so all bundles are loaded from the custom CDN.

### CODE REVIEW

**Review by dk1027 — APPROVED** [review]

File: template/snippet.js [hunk 1]
```diff
@@ -46,8 +46,8 @@
   // for methods in Analytics.js so that you never have to wait
   // for it to load to actually record data. The `method` is
   // stored as the first argument, so we can replay the data.
-  analytics.factory = function(method){
-    return function(){
+  analytics.factory = function(method) {
+    return function() {
```
Comment (juliofarah) [inline template/snippet.js:50]:
```suggestion
  analytics.factory = function(method){
    return function(){
```

File: template/snippet.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(function(){
+(function() {
```
Comment (juliofarah) [inline template/snippet.js:1]:
```suggestion
(function(){
```

File: template/snippet.js [hunk 3]
```diff
@@ -63,7 +63,7 @@
 
   // Define a method to load Analytics.js from our CDN,
   // and that will be sure to only ever load it once.
-  analytics.load = function(key, options){
+  analytics.load = function(key, options) {
```
Comment (juliofarah) [inline template/snippet.js:66]:
```suggestion
  analytics.load = function(key, options){
```

File: test/render.test.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ describe('snippet', function() {
 
     it('should set the ajs path', function() {
       assertContains(
-        snippet.max({ 
+        snippet.max({
```
Comment (juliofarah) [inline test/render.test.js:24]:
```suggestion
        snippet.max({ 
```

File: package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@segment/snippet",
   "author": "Segment.io <friends@segment.com>",
-  "version": "4.14.2",
+  "version": "4.15.0",
```
Comment (juliofarah) [inline package.json:4]: @pooyaj minor v ok?

**Review by danieljackins — APPROVED** [review]

File: package.json (hunk 5 above)
Comment (pooyaj) [reply package.json:4]: Yes 👍

**Review by pooyaj — APPROVED** [review]

**Pull request merged**